app/shared/components/Tools/KeyGenerator.js

```diff
@@ -219,11 +221,10 @@ class ToolsKeyGenerator extends Component<Props> {
               </Segment>
 
             </React.Fragment>
-          )
-          : (
```

Comment:
I've been intentionally formatting these conditional statements in JSX like this:

```
{(logic === true)
   ? ( if )
   : ( else )
}
```

So you can see the `?` and `:` at the start of the lines. I've noticed you correct them a few times, we should come up with what our standard is for this.

Reply:
Ok I see, that makes sense so I'll try to start doing that instead.